Automattic/jetpack
# Build: add missing mirror repo details, so package can be deployed

Follow-up from #22477

#### Changes proposed in this Pull Request:

* Add mirror repo info, so the package can be used in plugins such as Jetpack in released versions.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Click on the Build check below, and then access the summary view.
* click to download "jetpack-build"
* Uncompress the downloaded archive, and ensure there is a jetpack-plugins-installer directory in there.
